versioned_docs/version-2.x/magic-button/contribution-mode.mdx

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 1
+title: Contribution Mode
+description:
+  Contribution Mode is the core of Magic Button, streamlining server-client
+  interaction for collaborative and efficient content creation.
```

Comment:
**nitpick:** I don't find it to really express what the feature does.

**suggestion:** Contribution Mode is the core of the Magic Button tool. When Contribution Mode is enabled for a user, your storefront displays a set of tools created for content creators.